cmd/cephcsi.go

```diff
@@ -111,6 +111,8 @@ func main() {
 		fmt.Println("Go Version:", runtime.Version())
 		fmt.Println("Compiler:", runtime.Compiler)
 		fmt.Printf("Platform: %s/%s\n", runtime.GOOS, runtime.GOARCH)
+		kv, _ := util.KernelVersion()
```

Comment:
Error is discarded here.

Reply:
yes, would you prefer to log the error in case the kernel version can not be obtained?
I am considering to have the function never fail, and just return an empty string in case the kernel function can not be found (which should be extremely unlikely)

Reply:
i would prefer to log kernel version if err==nil instead of logging empty value

Reply:
ok, only logging the kernel version if it could get fetched